<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS Shell Dlg 2</family>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>untitled.pose - PoseDesigner</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Users/Clinton/Desktop/logo.png</normaloff>../../Users/Clinton/Desktop/logo.png</iconset>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" rowspan="6">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Pose Information</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>451</width>
            <height>842</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Samples</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QListWidget" name="listWidget">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="alternatingRowColors">
                <bool>false</bool>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Selected Sample Preview</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QGraphicsView" name="samplePreview">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>256</width>
                 <height>256</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Sample Statistics</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>522</height>
              </size>
             </property>
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="autoScroll">
              <bool>true</bool>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="sortingEnabled">
              <bool>true</bool>
             </property>
             <property name="rowCount">
              <number>16</number>
             </property>
             <property name="columnCount">
              <number>9</number>
             </property>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <row/>
             <column>
              <property name="text">
               <string>Left Shoulder</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Left Elbow</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Left Hip</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Left Knee</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Waist</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Right Shoulder</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Right Elbow</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Right Hip</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Right Knee</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="buttonTakeSample">
      <property name="toolTip">
       <string>Add New Sample</string>
      </property>
      <property name="statusTip">
       <string>Add New Sample</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Users/Clinton/Desktop/add.png</normaloff>../../Users/Clinton/Desktop/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="6">
     <widget class="GLWidget" name="kinectWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="buttonTakeSampleTimer">
      <property name="toolTip">
       <string>Set Timer</string>
      </property>
      <property name="statusTip">
       <string>Set Timer to Take Sample</string>
      </property>
      <property name="text">
       <string>Set Timer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Users/Clinton/Desktop/timer.png</normaloff>../../Users/Clinton/Desktop/timer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="buttonRemoveSample">
      <property name="toolTip">
       <string>Remove Selected Sample</string>
      </property>
      <property name="statusTip">
       <string>Remove Selected Sample</string>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Users/Clinton/Desktop/remove.png</normaloff>../../Users/Clinton/Desktop/remove.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QToolButton" name="buttonCalculate">
      <property name="toolTip">
       <string>Recalculate Sample Statistics</string>
      </property>
      <property name="statusTip">
       <string>Recalculate Sample Statistics</string>
      </property>
      <property name="text">
       <string>Recalculate</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Users/Clinton/Desktop/math.png</normaloff>../../Users/Clinton/Desktop/math.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QToolButton" name="buttonKinectSettings">
      <property name="toolTip">
       <string>Adjust Kinect Settings</string>
      </property>
      <property name="statusTip">
       <string>Adjust Kinect Settings</string>
      </property>
      <property name="text">
       <string>Kinect Settings</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../Users/Clinton/Desktop/kinect.png</normaloff>../../Users/Clinton/Desktop/kinect.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionCapture"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionPoseDesigner_Help"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_PoseDesigner"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <action name="actionCapture">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/open.png</normaloff>../../Users/Clinton/Desktop/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/saveas.png</normaloff>../../Users/Clinton/Desktop/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="toolTip">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/save.png</normaloff>../../Users/Clinton/Desktop/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/new.png</normaloff>../../Users/Clinton/Desktop/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/exit.png</normaloff>../../Users/Clinton/Desktop/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/undo.png</normaloff>../../Users/Clinton/Desktop/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/copy.png</normaloff>../../Users/Clinton/Desktop/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/cut.png</normaloff>../../Users/Clinton/Desktop/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/paste.png</normaloff>../../Users/Clinton/Desktop/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/delete.png</normaloff>../../Users/Clinton/Desktop/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/redo.png</normaloff>../../Users/Clinton/Desktop/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/selectall.png</normaloff>../../Users/Clinton/Desktop/selectall.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout_PoseDesigner">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/information.png</normaloff>../../Users/Clinton/Desktop/information.png</iconset>
   </property>
   <property name="text">
    <string>About PoseDesigner</string>
   </property>
   <property name="toolTip">
    <string>About PoseDesigner</string>
   </property>
  </action>
  <action name="actionPoseDesigner_Help">
   <property name="icon">
    <iconset>
     <normaloff>../../Users/Clinton/Desktop/question.png</normaloff>../../Users/Clinton/Desktop/question.png</iconset>
   </property>
   <property name="text">
    <string>PoseDesigner Help</string>
   </property>
   <property name="toolTip">
    <string>PoseDesigner Help</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
